.github/workflows/daily-tests.yml

```diff
@@ -133,20 +133,46 @@ jobs:
 
 
   feature-powerset:
-    name: Feature Powerset
-    runs-on: ubuntu-latest
+    # UPDATED TITLE IN THIS FORK
+    # FOR FUTURE CONSIDERATION: TRY WITH `cargo hack test`
+    name: Feature Powerset - build check
+    # FOR MORE FLEXIBLE FEATURE POWERSET CI CONFIGURATION IN THIS FORK
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        rust:
+          # MULTIPLE RUST VERSIONS FOR FUTURE CONSIDERATION IN THIS FORK
+          - stable
+        os:
+          # FEATURE POWERSET ON MULTIPLE PLATFORMS IN THIS FORK
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
```

Comment:
triggered daily build test: https://github.com/brody4hire/portable-rustls/actions/runs/13166334718

I noticed that the OS info is cut off in the jobs list here, want to fix this if possible.